src/cli/cmds/pod.toit

```diff
@@ -130,6 +130,90 @@ create-pod-commands -> List:
       --run=:: download it
   cmd.add download-cmd
 
+  tag-cmd := Command "tag"
+      --help="""
+        Change the tags of an uploaded pod.
+        """
+  cmd.add tag-cmd
+
+  tag-add-cmd := Command "add"
+      --help="""
+        Add tags to uploaded pods.
+
+        The pods to change are specified through a pod reference like
+        name@tag or name#revision.
+        If only a pod name is provided, the pod with the 'latest' tags is
+        changed.
+
+        If '--force' is provided, tags are added even if they already exists, moving
+        the tags to the selected pods.
+
+        Note: this operation is not atomic. It downloads the existing tags, adds the
+        new tags, and then sets the combined tags.
+        """
+      --options=[
+        Option "tag"
+            --short-name="t"
+            --help="A tag to add to the pod."
+            --multi,
+        Flag "force"
+            --short-name="f"
+            --help="Force tags even if they already exist."
+            --default=false,
+      ]
+      --rest=[
+        Option "reference"
+            --help="A pod reference: a UUID, name@tag, or name#revision."
```

Comment:
There can be more than one. Not sure how you usually write the help for such cases.

Reply:
Same.